content/sensu-go/5.18/guides/clustering.md

```diff
@@ -258,11 +258,44 @@ See [Secure Sensu][16] for information about cluster security.
 
 ## Use an external etcd cluster
 
-To use Sensu with an external etcd cluster, you must have etcd 3.3.2 or newer.
-To stand up an external etcd cluster, follow etcd's [clustering guide][2] using the same store configuration.
+You must have etcd 3.3.2 or newer to use Sensu with an external etcd cluster.
```

Comment:
```suggestion
Using Sensu with an external etcd cluster requires etcd 3.3.2 or newer, but is not compatible with etcd 3.4.0 or newer.
```

Reply:
It occurs to me that our datastore reference is focused on Postgres, but would also be a good place to document etcd details like this requirement.

Reply:
I've been running 3.4.3 since I built my cluster.  What is the failure in compatibility?

Reply:
@tarcinil I am not 100% sure but I think it is related to an outstanding etcd issue https://github.com/etcd-io/etcd/issues/11884

Reply:
> I've been running 3.4.3 since I built my cluster. What is the failure in compatibility?

It's compatible but we've seen a 50% performance loss and some instability at high load.